js/books.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+/* eslint-disable no-implied-eval */
+const book = new Book();
+book.loadPrev();
+
+document.getElementById('add_book').addEventListener('click', () => {
+  book.addBook();
+});
+
+function target(element) {
+  book.removeBook(element.id, element.value);
+}
+
+function displayC() {
+  const refresh = 1000; // Refresh rate in milli seconds
+  mytime = setTimeout('display_ct()', refresh);
+}
+
+function displayCt() {
+  const x = new Date();
+  const x1 = x.toUTCString();// changing the display to UTC string
+  document.getElementById('ct').innerHTML = x1;
+  tt = displayC();
+}
+
+displayCt();
+
```

Comment:
- [ ] While this is a good approach to displaying the current date and time, the project specifications state that you should use the [luxon](https://moment.github.io/luxon/#/install) library to achieve this. Kindly work on it as well